src/modules/manaFiatGateway/utils.ts

```diff
@@ -1,3 +1,31 @@
 import { channel } from 'redux-saga'
+import { ChainId, Network } from '@dcl/schemas'
+import { Environment } from './types'
 
 export const purchaseEventsChannel = channel()
+
+const chainIdsByEnvAndNetwork = {
+  [Environment.DEVELOPMENT]: {
+    [Network.ETHEREUM]: ChainId.ETHEREUM_GOERLI,
+    [Network.MATIC]: ChainId.MATIC_MUMBAI
+  },
+  [Environment.STAGING]: {
+    [Network.ETHEREUM]: ChainId.ETHEREUM_MAINNET,
+    [Network.MATIC]: ChainId.MATIC_MAINNET
+  },
+  [Environment.PRODUCTION]: {
+    [Network.ETHEREUM]: ChainId.ETHEREUM_MAINNET,
+    [Network.MATIC]: ChainId.MATIC_MAINNET
+  }
+}
+
+export const getChainIdByEnvAndNetwork = (
```

Comment:
Can't the [getChainIdByNetwork](https://github.com/decentraland/decentraland-dapps/blob/42bd2ff856b15b3678577d9598dec3b8032591af/src/lib/eth.ts#L105) function be used for this?